rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,624 @@
+        
+        wgMIds[wgData[uid]['prop_calc_wg'].model_id] = uid
+        wgMIds[wgData[uid]['prop_precalc_wg'].model_id] = uid
+        wgData[uid]['prop_calc_wg'].observe(handle_change, names='value')
+        wgData[uid]['prop_precalc_wg'].observe(handle_change, names='value')
+        
+        
+    # useDrawAs
+    if useDrawAs:
+        
+        if moldict is not None:
+            wgData[uid]['drawAs_wg'] = Dropdown(description='', options=DRAWING_LIGAND_3D, value=drawAs)
+            wgBox.append(Box([Label(value='drawAs'),wgData[uid]['drawAs_wg']], layout=itemLayout))
+            wgMIds[wgData[uid]['drawAs_wg'].model_id] = uid
+            wgData[uid]['drawAs_wg'].observe(handle_change, names='value')
+
+        if protein is not None:
+            wgData[uid]['pStyle_wg'] = Dropdown(description='', options=DRAWING_PROTEIN_3D, value=pStyle)
+            wgBox.append(Box([Label(value='pStyle'),wgData[uid]['pStyle_wg']], layout=itemLayout))
+            wgMIds[wgData[uid]['pStyle_wg'].model_id] = uid
+            wgData[uid]['pStyle_wg'].observe(handle_change, names='value')
+            
+            wgData[uid]['pStyle_tube_wg'] = Checkbox(description='helicesAsTubes', value=False)
+            wgBox.append(Box([Label(value=''),wgData[uid]['pStyle_tube_wg']], layout=itemLayout))
+            wgMIds[wgData[uid]['pStyle_tube_wg'].model_id] = uid
+            wgData[uid]['pStyle_tube_wg'].observe(handle_change, names='value')
+    
+    
+    # colors
+    if colorPanel is True:
+        wgData[uid]['colorScheme'] = Dropdown(description='', options=LIGAND_COLOR_SCHEME_3D, value='default')
+        wgBox.append(Box([Label(value='ligand color'),wgData[uid]['colorScheme']], layout=itemLayout))
+        wgMIds[wgData[uid]['colorScheme'].model_id] = uid
+        wgData[uid]['colorScheme'].observe(handle_change, names='value')
+        
+    # labels
+    if labelPanel is True:
+        wgData[uid]['confLabel'] = Checkbox(description='confLabel', value=False)
+        wgData[uid]['atomLabel'] = Checkbox(description='atomLabel', value=False)
+        
+        cbLabel=Box([Label(value=''),
+                     HBox([wgData[uid]['confLabel'], 
+                           wgData[uid]['atomLabel']])
+                    ], layout=itemLayout)
+        
+        wgBox.append(cbLabel)
+        
+        wgMIds[wgData[uid]['confLabel'].model_id] = uid
+        wgMIds[wgData[uid]['atomLabel'].model_id] = uid
+        wgData[uid]['confLabel'].observe(handle_change, names='value')
+        wgData[uid]['atomLabel'].observe(handle_change, names='value')
+    
+    
+    # background
+    wgData[uid]['background'] = Dropdown(description='', options= BGCOLORS_3D, value=BGCOLORS_3D[0])
+    wgBox.append(Box([Label(value='background'),wgData[uid]['background']], layout=itemLayout))
+    wgData[uid]['background'].observe(handle_change, names='value')
+    wgMIds[wgData[uid]['background'].model_id] = uid
```

Comment:
```
observers.append('background')
```

Now all the observers have been added, so the code that links `observe` with 'handle_change` can be moved here.